src/codegen/objects.rs

```diff
@@ -18,7 +18,7 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>, traits: &
         let class_analysis = match info {
             Some(info) => info,
             None => {
-                warn!("Class or interface {} not found.", obj.name);
+                info!("Class or interface {} not found.", obj.name);
```

Comment:
IMHO if config states that type must generated and no such type, then it sure warning and maybe ever error,
and better seen in run log and not when checking file list later.